docker/Dockerfile

```diff
@@ -16,8 +16,6 @@ RUN ./gradlew assemble
 
 FROM zookeeper:3.5.5
 COPY bin /usr/local/bin
+RUN apt update && apt install curl -y
 RUN chmod +x /usr/local/bin/*
 COPY --from=0 /zu/build/libs/zu.jar /root/
-
-RUN apt-get -q update && \
-    apt-get install -y dnsutils
```

Comment:
zookeeperStart is using `nslookup` - isn;t that provided by dnsutils?
https://github.com/pravega/zookeeper-operator/blob/20106e883c62d143eaa0c1c26fd7c59f0328dcc8/docker/bin/zookeeperStart.sh#L56

Reply:
Thanks fixing now.